lib/howitzer/web/page.rb

```diff
@@ -1,6 +1,7 @@
 require 'singleton'
 require 'rspec/expectations'
 require 'addressable/template'
+require 'howitzer/capybara_helpers'
```

Comment:
it is bad idea. This module contains helpers for config folder and contains a lot of extra methods which are not desired to Mixin to page object